business/MetadataDatabase/Models/SeriesContext.cs

```diff
@@ -0,0 +1,18 @@
+namespace MetadataDatabase.Models
+{
+    using MongoDB.Driver;
+    using System;
+
+    public class SeriesContext: IModelContext<Series>
+    {
+        private readonly IMongoDatabase _db;
```

Comment:
Weird indentation, no ?

Reply:
euh no SeriesContext extends of interface IModelContext with usage of types Series